### DISCUSSION TITLE
opensoundscape + soundevent package?

### DISCUSSION THREAD

**Discussion by paulpeyret-biophonia** [kitzeslab/opensoundscape#975] [General]
Hi there,

Following the discussion started [here ](https://github.com/kitzeslab/opensoundscape/issues/749#issuecomment-2051739207) with @sammlapp, I am wondering if you are willing to develop interoperability between opensoundscape package and the [soundevent](https://mbsantiago.github.io/soundevent/) package?

I can imagine the benefits of having the packages talking together and standardizing bioacoustics data structures. 🤝 
Is there some joined development effort between opensoundscape development team and the soundevent dev team or are these distinct projects? 

Thanks,
Paul

**Comment by sammlapp** [comment]
Hey @paulpeyret-biophonia, yes we have discussed integration with @mbsantiago of soundevent. It's not exactly clear to me yet how we will want to interface with soundevent in opensoundscape. If either of you have specific i/o functionality you would like to see, you can open an issue or a pull request.
